libafl/src/executors/forkserver.rs

```diff
@@ -878,6 +974,13 @@ impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> {
         self
     }
 
+    #[must_use]
+    /// Call this to set a defauult const coverage map size
+    pub fn coverage_map_size(mut self, size: usize) -> Self {
```

Comment:
We probably want to use `UnixShMem` instead of `StdShMem` - Std is not working for forkserver on MacOS (and potentially Android?)

Reply:
(It'd need fixes, the problem is that serializing the shmem information does not work across processes)

Reply:
`impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> `

this was already in place, we can change it but we have to check what AFL++ is using on macos

Reply:
UnixShMem is fine

Reply:
that's for another pr btw i think

Reply:
It would fix CI

Reply:
yes i will make the fuzzer linux only like forkserver simple

Reply:
Why?